src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -69,6 +70,20 @@ public AcquireTokenForClientParameterBuilder WithSendX5C(bool withSendX5C)
             return this;
         }
 
+        /// <summary>
```

Comment:
CC @jmprieur for new public API and comments

Reply:
Let's add a bit more context here, maybe:

```
If set, MSAL tries to auto-detect and use a regional Azure authority. This helps keep the authentication traffic inside the Azure region. If the region cannot be determined (e.g. not running on Azure), an exception is thrown.
```

Reply:
As per latest discussions, comments must be kept minimal and should indicate that this feature requires a server side configuration (which won't be available for a while)

Reply:
Updated.